--- Makefile.dist	2005-02-10 22:15:15.367051840 +0100
+++ Makefile	2005-02-10 22:15:57.632626496 +0100
@@ -1,7 +1,7 @@
 # Makefile for the pingtunnel utility
 # (c) 2004-2005 Daniel Stødle, daniels@stud.cs.uit.no
 
-CC	 		= gcc
+CC 		= gcc
 CFLAGS		= -Wall -g
 LDOPTS		= -lpthread -lpcap
 PT_OBJS		= ptunnel.o
@@ -17,8 +17,8 @@
 	mv PingTunnel/Makefile.dist PingTunnel/Makefile
 	
 
-install: ptunnel
-	install -D ./ptunnel $(bindir)/ptunnel
+install: 
+	install -D ./ptunnel $(DESTDIR)$(bindir)/ptunnel
 
 ptunnel: $(PT_OBJS)
 	$(CC) $(LDOPTS) -o $@ $^
